src/js/builtins/ProcessObjectInternals.ts

```diff
@@ -59,11 +53,10 @@ export function getStdioWriteStream(fd) {
     };
   }
 
-  stream._type = "tty";
   stream._isStdio = true;
   stream.fd = fd;
 
-  return stream;
+  return [stream, stream[require("internal/shared").fileSinkSymbol]];
```

Comment:
Returning the stream plus a reference to the symbol seems kind of hacky. Instead can we just expose the symbol to  C++ separately.